specifications/api/swagger.yaml

```diff
@@ -3137,7 +3250,46 @@ components:
         metadata: *transactionMetadata
         time_to_live: *transactionTTL
 
+    ApiTransactionExtended: &ApiTransactionExtended
+      type: object
+      required:
+        - cborHex
+        - description
+        - type
+      properties:
+        cborHex: *serialisedTransactionHex
+        description: *txDescription
+        type: *txType
```

Comment:
Is this precision needed?

When submitting txs through the node, we don't have to specify the era. Why here?

Reply:
In this place I only complied with what plutus scripts have inside. See examples in shelley/test/data/plutus directory of this PR